features/user-applies-approved-edits-co.feature

```diff
@@ -36,8 +38,10 @@ Feature: Whole process of a new user creating a login, applying, being approved,
       | shf_applications.new.company_number | shf_applications.new.phone_number | shf_applications.new.contact_email |
       | 5560360793                          | 031-1234567                       | new_user@example.com               |
     And I select "Groomer" Category
+    And I select files delivery radio button "upload_now"
```

Comment:
If the files are not going to actually be uploaded during the scenario, don't chose this option. How about "upload later" instead?